frontend/client-search/client-search-input.component.tsx

```diff
@@ -45,7 +46,12 @@ export default function ClientSearchInput(props: ClientSearchInputProps) {
   }
 
   function getInitialSearch(): Search {
-    const value = props.initialValue || "";
+    let value: string;
+    if (props.initialValueFromQueryParams) {
+      value = deserializeSearch();
```

Comment:
This change allows for the search text input to start out with the value in the query params.